JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -104,10 +104,15 @@ module Microsoft.ApplicationInsights {
             }
 
             // enable session storage buffer experiment
-            this.config.enableSessionStorageBuffer = new SplitTest().isEnabled(this.config.instrumentationKey, 10); ;
+            var enableExperiment = new SplitTest().isEnabled(this.config.instrumentationKey, 10);
+            this.config.enableSessionStorageBuffer = enableExperiment;
 
             this.context = new ApplicationInsights.TelemetryContext(configGetters);
 
+            DataLossAnalyzer.appInsights = this;
```

Comment:
I agree. I'll create an issue and refactor DataLossAnalyzer in a separate pull request. Thanks!